docs/cache-layer-guide.md

```
@@ -0,0 +1,249 @@
+    await this.cacheService.delByPattern(`posts:${command.userId}:*`);
+  }
+}
+```
+
+## 로깅
+
+`CacheService`는 NestJS `Logger`를 내장하며, 모든 연산을 `debug` 레벨로 로깅한다.
+
+### 로그 출력 예시
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)